cmake/pcl_options.cmake

```diff
@@ -64,3 +64,14 @@ set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 option(BUILD_tools "Useful PCL-based command line tools" ON)
 
 option(WITH_DOCS "Build doxygen documentation" OFF)
+
+# set index size
+set(PCL_INDEX_SIZE -1 CACHE STRING "Set index size. Available options are: 8 16 32 64")
```

Comment:
The help string should probably clarify to the user what's the meaning of -1 or in general negative values.

Reply:
Should the type be STRING? I don't know if CMake has non-string types

Reply:
The available option are relatively limited https://cmake.org/cmake/help/latest/command/set.html#set-cache-entry

Reply:
Ignore me 🙈